setup.py

```diff
@@ -1,5 +1,12 @@
+import sys
 from setuptools import setup, find_packages
 
+if sys.version_info < (3,7):
+    sys.exit('Sorry, Python < 3.7 is not supported, upgrade to >=3.7, <3.8')
+
+if sys.version_info > (3,7):
+    sys.exit('Sorry, Python > 3.7 is not supported, downgrade to >=3.7, <3.8')
```

Comment:
Python 3.7 is just required if users want to use the package to submit agents to kaggle using our framework. Some users may be just using the python game environment for their own use-cases not using the RL agent framework we've added.

Maybe just add a warning message in this case? Maybe also add a warning to the train.py script that the models won't work in submissions unless 3.7 is used?

Reply:
Sounds good, gimme a moment :+1:

Reply:
Ok, this should work both on linux and windows.